elfpy/data/acquire_data.py

```diff
@@ -103,19 +115,18 @@ def main(
                         continue
 
                 if block_pool_info:
-                    postgres.add_pool_infos([block_pool_info], session)
                     pool_info.append(block_pool_info)
+
```

Comment:
pls try to avoid extraneous newlines, per [pep8](https://peps.python.org/pep-0008/#:~:text=Extra%20blank%20lines,indicate%20logical%20sections) they should be used sparingly to differentiate logical sections. We've been using commented lines to serve this purpose instead, since they explain the different sections and also serve as a visual break.